registry/remote/repository.go

```diff
@@ -1453,7 +1453,11 @@ func (s *manifestStore) updateReferrersIndex(ctx context.Context, subject ocispe
 		}
 
 		// 3. push the updated referrers list using referrers tag schema
-		if len(updatedReferrers) > 0 {
+		if len(updatedReferrers) > 0 || s.repo.SkipReferrersGC {
+			// push a new index when:
+			// 1. updatedReferrers is not empty
+			// 2. OR referrers GC is skipped, in this case the old index
+			//    won't get deleted and an empty index should be pushed
```

Comment:
This block of comment is a bit confusing. You may reword it to be more natural.

Reply:
Reworded